Altis_Life.Altis/stringtable.xml

```diff
@@ -11175,6 +11175,17 @@
             <Russian>Вы были обнаружены для мошенничества и было сообщено на сервер. Удачного дня.</Russian>
             <German>Sie wurden für Betrug erkannt und haben an den Server gemeldet. Genieße deinen Tag.</German>
```

Comment:
@setoy, is the [above translation](https://github.com/AsYetUntitled/Framework/pull/120#discussion_r87704972) more accurate than the current one?